src/main/java/com/ibm/cloud/sdk/core/security/IamAuthenticator.java

```diff
@@ -361,13 +348,14 @@ protected void init(String apikey, String url, String clientId, String clientSec
 
   @Override
   public void validate() {
+    super.validate();
```

Comment:
The new base class has a validate function that validates the common fields contained in that class.